login.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Wolox Login</title>
+  <meta name="description" content="training, login card">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body class="body--white">
+  <div class="container">
+    <div class="login__wrapper">
+      <div class="login__card">
+        <div class="login__logo"></div>
+        <form id="loginForm" action="./list.html">
+          <label class="login__label" for="email">Email</label>
+          <input class="login__input" type="email" id="email" required>
+          <label class="login__label" for="password">Password</label>
+          <input class="login__input" type="password" id="password" password required>
+          <button class="login__btn" id="btnSubmit" login>Login</button>
+        </form>
+        <hr class="divider">
```

Comment:
We like to see these dividers as decorative elements. You can even have a class that add this style with a pseudo element.

Reply:
great!